content/en/docs/howto/charts_tips_and_tricks.md

```diff
@@ -224,6 +224,10 @@ spec:
 [...]
 ```
 
+Each invocation of the template will generate a unique random string. This
```

Comment:
Where it says "template" I think you mean the "template function" `randAlphaNum`. It might be worth while to call it a template function.

Reply:
Thanks, @mattfarina - please see the update and let me know what you think.

Reply:
Maybe we should clarify users would need to set a variable once to reuse the same random string even in the same file. Something like:

```yaml
{{ $random := randAlphaNum 5 | quote }}
---
kind: Deployment
spec:
  template:
    metadata:
      annotations:
        example: {{ $random }}
---
kind: Service
metadata:
  labels:
    example: {{ $random }}
```

Reply:
It's also possible to generate a random value once and then reuse it across numerous templates. Should we document that, too? Maybe as follow-ups

Reply:
> It's also possible to generate a random value once and then reuse it across numerous templates. Should we document that, too? Maybe as follow-ups

How?